docs/release/trg-0/TRG 6.02 API Versioning.md

```diff
@@ -0,0 +1,62 @@
+---
+title: TRG 6.02 - API Versioning
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: "mandatory after": 01st of May 2024
+:::
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 30-Nov-2023  | Initial contribution                   |
+
+## Why
+
+For publicly available APIs a versioning mechanism is required, so that all APIs are versioned the same way.
+
+Note that for APIs accessible only via EDC, the API-version is included in the EDC asset and not the URL. This is described in CX-0018.
+
+## Description
+
+For public REST-APIs (= not hidden behind an EDC aspect), the Version number **must** be included in the URL.
+
+Example:  `https://example.com/api/v2/customers`
+
+A standard **should** offer the following aliases as well (meaning they all answer identically to HTTP requests):
+
+`https://example.com/api/v2.1/customers`
+
+`https://example.com/api/latest/customers`
+
+For the version in the URL, semantic versioning **must** be used.
+
+Old endpoints (that have been offered bevor) and are not supported anymore **must** respond with an appropriate HTTP 30x status code and the new location (URL).

```

Comment:
I personally would still prefer a dedicated TRG. We could still add a small section in this one and additionally link to the other TRG. Like I mentioned, i think this makes it easier to maintian and also easier to understand and follow.

But as also said, this is my personal opinion and we should discuss this with more people, since if this will be released as TRG, everyone has to comply to it.
We could of course also extract it to a dedicated TRG later on